frontend/src/lib/components/icons.tsx

```diff
@@ -1415,3 +1415,65 @@ export function IconUploadFile(props: React.SVGProps<SVGSVGElement>): JSX.Elemen
         </svg>
     )
 }
+
+export function CheckCircleOutlined(props: React.SVGProps<SVGSVGElement>): JSX.Element {
```

Comment:
This icon naming convention absolutely makes sense, but we've already got one, so this should be named `IconCheckRicle` (outlined is the default variant for us)

Reply:
I merged in my other PR with a `IconCheckCircleOutline` so you can use that! (if it looks to be the same one)